test/solidity/block/BlockInfo.js

```diff
@@ -0,0 +1,105 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+const { BigNumber } = require('ethers');
+const Constants = require('../../constants')
+
+describe('BlockInfo Test Suite', function () {
```

Comment:
Let's add a tag for easy running identification. Maybe
```suggestion
describe('@solidityevmequiv BlockInfo Test Suite', function () {
```